src/components/Pagination/Pagination.tsx

```diff
@@ -356,7 +373,9 @@ export const Pagination: FC<PaginationProps> = React.forwardRef(
                                                 ])}
                                                 shape={ButtonShape.Rectangle}
                                                 key="previous"
-                                                disabled={_currentPage <= 1}
+                                                disabled={
+                                                    !loop && _currentPage <= 1
```

Comment:
whats the "less than" scenario?

Reply:
I figured it's better to be safe than sorry in a 0 index scenario. leaving as is.

Reply:
but... if it happened to be zero indexed this would be a bug and disable the button prematurely, right?

Reply:
Current implementation doesn't allow for a zero index as it would be reflected in the buttons [0], [1], [2] ... leaving as is for now until a refactor considers 0 index.